samsung-galaxybook.c

```diff
@@ -1933,7 +1933,7 @@ static struct platform_driver galaxybook_platform_driver = {
 		.acpi_match_table = galaxybook_device_ids,
 	},
 	.probe = galaxybook_probe,
-	.remove_new = galaxybook_remove,
+	.remove = galaxybook_remove,
```

Comment:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 13, 0)
	.remove = galaxybook_remove,
#else
	.remove_new = galaxybook_remove,
#endif
```
This is going to be fixed when the driver is mainlined (in progress now) but I am ok to add this for now here provided that it does not break the 99% of users running < 6.13.0 :slightly_smiling_face: 
This should probably do it? But please feel free to test!

Reply:
Hi,

I saw this too and was wondering how to make the change without breaking the existing. I've tested your solution on my PC (kernel 6.13.5), and it's working for me.
Thanks for your work!

Have a nice day.